server/text_generation_server/utils/hub.py

```diff
@@ -138,33 +179,33 @@ def download_weights(
 ) -> List[Path]:
     """Download the safetensors files from the hub"""
 
-    def download_file(filename, tries=5, backoff: int = 5):
-        local_file = try_to_load_from_cache(model_id, revision, filename)
+    def download_file(fname, tries=5, backoff: int = 5):
```

Comment:
Note: these var renaming (filename -> fname, start_time -> stime and i -> idx) are not mandatory, just renamed them to avoid pep warnings like "shadows var_name from outer scope"